<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportSettings</class>
 <widget class="QDialog" name="ImportSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Import road types</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="motorway_cb">
          <property name="text">
           <string>motorway</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="secondary_cb">
          <property name="text">
           <string>secondary</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="primary_cb">
          <property name="text">
           <string>primary</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tertiary_cb">
          <property name="text">
           <string>tertiary</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="service_cb">
          <property name="text">
           <string>service</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="path_cb">
          <property name="text">
           <string>path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="steps_cb">
          <property name="text">
           <string>steps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="track_cb">
          <property name="text">
           <string>track</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="footway_cb">
          <property name="text">
           <string>footway</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="residential_cb">
          <property name="text">
           <string>residential</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="living_street_cb">
          <property name="text">
           <string>living_street</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cycleway_cb">
          <property name="text">
           <string>cycleway</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="turning_circle_cb">
          <property name="text">
           <string>turning_circle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pedestrian_cb">
          <property name="text">
           <string>pedestrian</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="unclassified_cb">
          <property name="text">
           <string>unclassified</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="maximize_curve_radius_cb">
          <property name="text">
           <string>maximize curve radius</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>LinearError:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="LinearErrorSpin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>CurveError:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="CurveErrorSpin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
